stdlib/tkinter/simpledialog.pyi

```diff
@@ -22,6 +25,30 @@ class SimpleDialog:
     def wm_delete_window(self) -> None: ...
     def done(self, num: int) -> None: ...
 
-def askfloat(title: str | None, prompt: str, **kwargs: Any) -> float | None: ...
-def askinteger(title: str | None, prompt: str, **kwargs: Any) -> int | None: ...
-def askstring(title: str | None, prompt: str, **kwargs: Any) -> str | None: ...
+def askfloat(
+    title: str | None,
+    prompt: str,
+    *,
+    initialvalue: float | None = ...,
+    minvalue: float | None = ...,
+    maxvalue: float | None = ...,
+    parent: Misc | None = ...,
+) -> float | None: ...
+def askinteger(
+    title: str | None,
+    prompt: str,
+    *,
+    initialvalue: int | None = ...,
+    minvalue: float | None = ...,  # Any object that can compare to int.
```

Comment:
It might be better to require `int`, because the value is also used in error messages, and displaying a float in an error message would be confusing to users.